### ISSUE TITLE
Mini-Gun Soldier weird behavior and others

### ISSUE THREAD

**Issue by Deadza** [pa1nki113r/Project_Brutality#355]
Hello, while playing I noticed the Mini Gun soldier variant is shooting platforms and elevators. I'm not sure what exactly triggers this behavior but it seems to be in-fighting with other enemies? I wish I was any help beyond that but it does keep happening. 

Also as a small aside I think the mini gun has way too much recoil now, just a preference thing I suppose but very hard to aim whatsoever making it regulated to fighting big ol' enemies or using in super close quarters mostly.. Another user issue of preference I suppose is the Demon Tech rifle now makes a huge and obscuring cloud effect when you shoot it, making it hard to aim as well because I can't really see what I'm shooting at. 

Thanks for all the hard work and sharing your efforts to the people compiling and editing the mod of course too.

**Comment by Deadza** [comment]
also as I play a few hours more I notice Pink Demons and their sub types all sorta just circle around and don't run at or toward the player. It happens sometimes with axe science zombies too.

**Comment by JMartinez9820** [comment]
@Deadza Care to explain the weird monster behavior? If you are using an old test build, this report will not be logged.

**Comment by Deadza** [comment]
no, all these issues were on the up to date (at the time) Master file when i reported and cant exactly describe it more. just exactly as i said. its weird interactions with light possibly (no texture mods) but i cant be sure and not moving directly at the player which sometimes caused endless circle loops. 

i will say the older test build im using now STILL has some of the wonky melee enemy behavior, though it seems a less intense and so i keep using it. in the test build melee enemies will all too often slow themselves down moving in a predictable zig zag pattern but never going full blown circles like the Master file did. ive compared this to other non Project Brutality mods (like BDoom) and the melee enemies just run toward the player pathing straight toward you just fine, never getting weird and possibly conflicting behaviors maybe. again not sure why it happens. 

the old file ive moved to however does not have the zombie men shooting at raised platforms, walls, whatever it is. i havent seen it in 20 some odd hours of using it, so it has to be something around the time of the Master i was using. 

again, i use no mods i can find a conflict with. Maps of Chaos and Music paks. And the behavior issues persist even if I remove Maps of Chaos. 

thanks

**Comment by goldenmob72** [comment]
Have you checked that on the current WIP Build @Deadza ? if it reaccurs on that build, Please let us know

**Comment by Deadza** [comment]
i tried a pretty recent update that happened i believe a day after my issues started? i still saw bizarre attacks on objects and at least the zig zag running. i didnt stick long to see anything else.

however i will redownload the now current Master and see if it all the behaviors still persist, give it a first map test or so.
